src/com.company/backtracking/SudokuSolver.java

```diff
@@ -0,0 +1,83 @@
+package backtracking;
+
+public class SudokuSolver {
+    public boolean solveSudoku(int[][] board) {
+        return canBeSolved(board, 0, 0);
+    }
+
+    private boolean canBeSolved(int[][] board, int row, int col) {
+        if(row == board.length) {
+            return true;
+        }
+        if(col == board[row].length) {
+            return canBeSolved(board, row + 1, 0);
+        }
+        if(board[row][col] != 0) {
+            return canBeSolved(board, row, col + 1);
+        }
+
+        for(int i = 1;  i<=9; i++) {
```

Comment:
Provide proper alignment. Use command + shift + enter.

Reply:
formatted the code